charpick/properties.c

```diff
@@ -18,557 +18,723 @@
 void
 register_stock_for_edit (void)
 {
-  static gboolean registered = FALSE;
-  if (!registered)
-  {
-    GtkIconFactory *factory;
-    GtkIconSet     *icons;
-
-    static const GtkStockItem edit_item [] = {
-           { CHARPICK_STOCK_EDIT, N_("_Edit"), 0, 0, GETTEXT_PACKAGE },
-    };
-    icons = gtk_icon_factory_lookup_default ("gtk-preferences");
-    factory = gtk_icon_factory_new ();
-    gtk_icon_factory_add (factory, CHARPICK_STOCK_EDIT, icons);
-    gtk_icon_factory_add_default (factory);
-    gtk_stock_add_static (edit_item, 1);
-    registered = TRUE;
-  }
+    static gboolean registered = FALSE;
+    if (!registered)
+    {
+        GtkIconFactory *factory;
+        GtkIconSet     *icons;
+
+        static const GtkStockItem edit_item [] = {
+            { CHARPICK_STOCK_EDIT, N_("_Edit"), 0, 0, GETTEXT_PACKAGE },
+        };
+        icons = gtk_icon_factory_lookup_default ("gtk-preferences");
+        factory = gtk_icon_factory_new ();
+        gtk_icon_factory_add (factory,
+                              CHARPICK_STOCK_EDIT,
+                              icons);
+        gtk_icon_factory_add_default (factory);
+        gtk_stock_add_static (edit_item, 1);
+        registered = TRUE;
+    }
 }
 
 #if 0
 static void
 set_atk_relation (GtkWidget *label, GtkWidget *widget)
 {
-  AtkObject *atk_widget;
-  AtkObject *atk_label;
-  AtkRelationSet *relation_set;
-  AtkRelation *relation;
-  AtkObject *targets[1];
-
-  atk_widget = gtk_widget_get_accessible (widget);
-  atk_label = gtk_widget_get_accessible (label);
-  
-  /* set label-for relation */
-  gtk_label_set_mnemonic_widget (GTK_LABEL (label), widget);	
-
-  /* return if gail is not loaded */
-  if (GTK_IS_ACCESSIBLE (atk_widget) == FALSE)
-    return;
-
-  /* set label-by relation */
-  relation_set = atk_object_ref_relation_set (atk_widget);
-  targets[0] = atk_label;
-  relation = atk_relation_new (targets, 1, ATK_RELATION_LABELLED_BY);
-  atk_relation_set_add (relation_set, relation);
-  g_object_unref (G_OBJECT (relation)); 
+    AtkObject *atk_widget;
+    AtkObject *atk_label;
+    AtkRelationSet *relation_set;
+    AtkRelation *relation;
+    AtkObject *targets[1];
+
+    atk_widget = gtk_widget_get_accessible (widget);
+    atk_label = gtk_widget_get_accessible (label);
+
+    /* set label-for relation */
+    gtk_label_set_mnemonic_widget (GTK_LABEL (label), widget);
+
+    /* return if gail is not loaded */
+    if (GTK_IS_ACCESSIBLE (atk_widget) == FALSE)
+        return;
+
+    /* set label-by relation */
+    relation_set = atk_object_ref_relation_set (atk_widget);
+    targets[0] = atk_label;
+    relation = atk_relation_new (targets, 1, ATK_RELATION_LABELLED_BY);
+    atk_relation_set_add (relation_set, relation);
+    g_object_unref (G_OBJECT (relation)); 
 }
 #endif /* 0 */
```

Comment:
this block can be removed

Reply:
You mean all from `#if 0` and `#endif /* 0 */`?

Reply:
Yes, the lines of code above can be removed because they are not currently in use and are marked as dead code.
The internal function is not used anywhere either:
```C
$ git grep -n set_atk_relation
charpick/properties.c:41:set_atk_relation (GtkWidget *label, GtkWidget *widget)
```

Reply:
Done